### PR TITLE
Simple fix for #58

### PR DESCRIPTION
**Pull request by commjoen**
Fix for #58 with:
- a more complete sample_attributes.yml file
- a better inspec.yml file so that it can work out of the box again
- a better formatted README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,76 +17,60 @@ InSpec is an open-source run-time framework and rule language used to specify co
...
+* `daemon_tlskey: /etc/docker/ssl/server_key.pem`
+  configure the server key.
+* `swarm_mode: inactive`
+  configure the swarm mode.
+* `swarm_max_manager_nodes: 3`
+  configure the maximum number of swarm leaders.
+* `swarm_port: 2377`
+  configure the swarm port.
+* `benchmark_version`
+  to execute also the old controls from previous benchmarks, e.g. set it to 1.12.0 to execute also the tests from cis-benchmark-1.12.0
```
Comment (chris-rock) [inline README.md]: Should we mention here that  1.12.0 is the default?

File: README.md [hunk 2]
```diff
@@ -17,76 +17,60 @@ InSpec is an open-source run-time framework and rule language used to specify co
...
+
+These settings can be overriden using an attributes file (e.g. --attrs <attributefile.yml>). See [sample_attributes.yml](sample_attributes.yml) as an example.
```
Comment (chris-rock) [inline README.md]: +1

File: inspec.yml [hunk 3]
```diff
@@ -17,6 +17,7 @@ attributes:
     required: true
     description: 'define needed capabilities for containers.'
     type: string
+    default: NET_ADMIN,SYS_ADMIN
```
Comment (chris-rock) [inline inspec.yml]: Great addition

File: sample_attributes.yml [hunk 4]
```diff
@@ -4,9 +4,10 @@ registry_cert_path: /etc/docker/certs.d
 registry_name: /etc/docker/certs.d/registry_hostname:port
 registry_ca_file: /etc/docker/certs.d/registry_hostname:port/ca.crt
 container_user: vagrant
-container_capadd: null
+container_capadd: 'NET_ADMIN,SYS_ADMIN'
```
Comment (chris-rock) [inline sample_attributes.yml]: Should the example be different from the default? E.g just use 'NET_ADMIN'?

Comment (commjoen) [reply sample_attributes.yml]: nice idea, done!

File: inspec.yml (hunk 3 above)
Comment (commjoen) [reply inspec.yml]: Thanks 👍

File: README.md (hunk 2 above)
Comment (commjoen) [reply README.md]: Thanks 👍

File: README.md (hunk 1 above)
Comment (commjoen) [reply README.md]: Done 👍

**Pull request merged**